core/pfe-core/controllers/tabs-aria-controller.ts

```diff
@@ -0,0 +1,123 @@
+import type { ReactiveController, ReactiveControllerHost } from 'lit';
+
+import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
+
+export interface TabsControllerOptions<Tab, Panel> {
```

Comment:
nit:  might consider renaming options to `TabsAriaControllerOptions` for consistency

Reply:
2515c32f